modules/imgproc/test/test_contours_new.cpp

```diff
@@ -89,22 +92,20 @@ TEST_P(Imgproc_FindContours_Modes1, rectangle)
         const Size sz(rng.uniform(640, 1920), rng.uniform(480, 1080));
         Mat img(sz, CV_8UC1, Scalar::all(0));
         Mat img32s(sz, CV_32SC1, Scalar::all(0));
-        const Rect r(
-            Point(rng.uniform(1, sz.width / 2 - 1), rng.uniform(1, sz.height / 2)),
-            Point(rng.uniform(sz.width / 2 - 1, sz.width - 1), rng.uniform(sz.height / 2 - 1, sz.height - 1)));
+        const Rect r(Point(rng.uniform(1, sz.width / 2 - 1), rng.uniform(1, sz.height / 2)),
+                     Point(rng.uniform(sz.width / 2 - 1, sz.width - 1),
+                           rng.uniform(sz.height / 2 - 1, sz.height - 1)));
         rectangle(img, r, Scalar::all(255));
         rectangle(img32s, r, Scalar::all(255), FILLED);
 
-        const vector<Point> ext_ref {
-            r.tl(),
-            r.tl() + Point(0, r.height - 1),
-            r.br() + Point(-1, -1),
-            r.tl() + Point(r.width - 1, 0)
-        };
+        const vector<Point> ext_ref {r.tl(),
+                                     r.tl() + Point(0, r.height - 1),
+                                     r.br() + Point(-1, -1),
+                                     r.tl() + Point(r.width - 1, 0)};
```

Comment:
Previous variant is better.

Reply:
Seems to be improved by setting `PenaltyIndentedWhitespace: 1`